### PR TITLE
Update remote-debugging-azure-app-service.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
MS style guide and less is more

### CODE REVIEW

**Comment by Court72** [comment]
@Mikejo5000

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by Mikejo5000 — APPROVED** [review]

**Comment by Mikejo5000** [comment]
#sign-off

**Comment by Rick-Anderson** [comment]
Thanks @Mikejo5000

File: docs/debugger/remote-debugging-azure-app-service.md [hunk 1]
```diff
@@ -45,16 +45,14 @@ Your app service instance now supports remote debugging through Visual Studio.
 
 ## Configure debugging settings
 
-Make sure you have completed the following steps in Visual Studio before debugging an app in Azure to ensure success.
-
-1. First, make sure you have successfully built your project at least one time. A successful build ensures the source code and any necessary compiled files are ready to go. If your application is running locally, make sure to stop the app.
+1. Build your app without errors. Stop your app if is running locally.
```
Comment (Court72) [inline docs/debugger/remote-debugging-azure-app-service.md:48]:
```suggestion
1. Build your app without errors. Stop your app if it's running locally.
```

**Pull request merged**